src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -359,17 +368,12 @@ internal static IVisualElementRenderer CreateRenderer(VisualElement element, Con
 			}
 
 			renderer.SetElement(element);
-			return renderer;
-		}
-
-		internal static IVisualElementRenderer CreateRenderer(VisualElement element, AndroidX.Fragment.App.FragmentManager fragmentManager, Context context)
```

Comment:
nothing exciting here. Only one API in Forms was using this so I just moved it all into the same `CreateRenderer `method